prog/test/github_runner.rb

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+require "octokit"
+require "yaml"
+
+class Prog::Test::GithubRunner < Prog::Test::Base
+  FAIL_CONCLUSIONS = ["action_required", "cancelled", "failure", "skipped", "stale", "timed_out"]
+  IN_PROGRESS_CONCLUSIONS = ["in_progress", "queued", "requested", "waiting", "pending", "neutral"]
+
+  def self.assemble(vm_host_id, test_cases)
+    github_service_project = Project.create(name: "Github Runner Service Project") { _1.id = Config.github_runner_service_project_id }
+    github_service_project.associate_with_project(github_service_project)
+
+    github_test_project = Project.create_with_id(name: "Github Runner Test Project")
+    GithubInstallation.create_with_id(
+      installation_id: Config.github_test_installation_id,
+      name: "GithubTestInstallationName",
+      type: "GithubTestInstallationType",
+      project_id: github_test_project.id
+    )
+    github_test_project.associate_with_project(github_test_project)
```

Comment:
nit: Associate after creation

```suggestion
    github_test_project = Project.create_with_id(name: "Github Runner Test Project")
    github_test_project.associate_with_project(github_test_project)
    GithubInstallation.create_with_id(
      installation_id: Config.github_test_installation_id,
      name: "GithubTestInstallationName",
      type: "GithubTestInstallationType",
      project_id: github_test_project.id
    )
```